packages/deployment/app/cloudformation.template.yml

```diff
@@ -1098,13 +1164,123 @@ Resources:
     Properties:
       Cluster:
         Ref: CumulusECSCluster
-      DesiredCount: {{# if this.count}}{{this.count}}{{ else }} 0 {{/if}}
+      DesiredCount: {{# if this.count}}{{this.count}}{{ else }} 1 {{/if}}
       TaskDefinition:
         Ref: {{@key}}TaskDefinition
       DeploymentConfiguration:
         MaximumPercent: 100
         MinimumHealthyPercent: 0
 
+  {{@key}}ECSServiceScalableTarget:
+    Type: AWS::ApplicationAutoScaling::ScalableTarget
+    DependsOn:
+    - {{@key}}ECSService
+    Properties:
+      MaxCapacity: {{# if this.max_count}}{{this.max_count}}{{ else }} 100 {{/if}}
+      MinCapacity: {{# if this.count}}{{this.count}}{{ else }} 1 {{/if}}
+      ResourceId:
+        Fn::Join:
+          - /
+          - - service
+            - Ref: CumulusECSCluster
+            - Fn::GetAtt:
+              - {{@key}}ECSService
+              - Name
+      RoleARN: {{../this.iams.scalingRoleArn}}
+      ScalableDimension: 'ecs:service:DesiredCount'
+      ServiceNamespace: ecs
+
+  {{@key}}ScaleOutScalingPolicy:
+    Type: "AWS::ApplicationAutoScaling::ScalingPolicy"
+    Properties:
+      PolicyName: {{@key}}ScaleOutScalingPolicy
+      ScalingTargetId:
+        Ref: {{@key}}ECSServiceScalableTarget
+      # Note: SimpleScaling is not supported by ApplicationAutoScaling
+      PolicyType: StepScaling
+      StepScalingPolicyConfiguration:
+      {{#if this.service_scaling.scale_out.step_adjustment_config}}
+        {{{ToJson this.service_scaling.scale_out.step_adjustment_config}}}
+      {{else}}
+        # Scaling out only happens once per activities waiting alarm is triggered (so assumes new activities are scheduled which can't be placed).
+        AdjustmentType: PercentChangeInCapacity
+        StepAdjustments:
+        - MetricIntervalLowerBound: 0
+          ScalingAdjustment: 10
+      {{/if}}
+
+  {{@key}}ActivitiesWaitingAlarm:
```

Comment:
I don't see this actually referenced anywhere else.  Am I missing something?  Is it not the default?

Reply:
Do you expect to see this alarm referenced somewhere else? The alarm is a standalone resource which is associated with `AlarmActions` which references the ScaleOutScalingPolicy.